src/server/Server.js

```diff
@@ -55,17 +55,19 @@ export default class Server {
     }
 
     connectionMade(socket) {
-        console.log(`[Server] Connection from: ${socket.id}`)
+        let ip = (socket.client.request.headers['cf-connecting-ip']) ? socket.client.request.headers['cf-connecting-ip'] : socket.request.connection.remoteAddress
```

Comment:
Could this result in IP spoofing if the CPPS is not using cloudflare? Like, for example, sending a request with the `cf-connecting-ip` header?

Reply:
Yes, it could, if a request is sent not through Cloudflare any headers can easily be spoofed.
Correct me if I'm mistaken but I don't think that's too much of a big deal though?
It could be used to circumvent IP bans, but there's no IP banning system in Yukon, so that would be the responsibility of individual servers to look into if they implement that feature.

Reply:
https://thisinterestsme.com/php-ip-address-cloudflare/

This could be fixed with this method if it is a big issue though